debian/changelog

```diff
@@ -1,3 +1,9 @@
+system76-driver (20.04.61~~alpha) focal; urgency=low
+
+  * Daily WIP for 20.04.61
```

Comment:
Need to add an entry

Reply:
"Daily WIP" will be removed when making a release, making the changelog invalid.

You need to add an entry describing the change, but don't remove the "Daily WIP" line.

Reply:
Oh, gotcha. I think it's better now